FitTracker/app/src/main/java/com/pps/globant/fittracker/mvp/model/LoginModel.java

```diff
@@ -1,4 +1,17 @@
 package com.pps.globant.fittracker.mvp.model;
 
+import com.pps.globant.fittracker.utils.FacebookLoginProvider;
+import com.squareup.otto.Bus;
+
 public class LoginModel {
+    private final Bus bus;
+
+    public LoginModel(Bus bus) {
+        this.bus = bus;
+        FacebookLoginProvider.registerCallback(bus);
```

Comment:
move to a new method, and call since presenter. This simplify the test, and mantain the logic into presenter